spec/validators/smrt_link_options_validator_spec.rb

```diff
@@ -26,7 +27,8 @@
     end
 
     context 'invalid' do
-      let(:plate) { create(:pacbio_plate, run: create(:pacbio_run, smrt_link_version: versions.first)) }
+      let!(:run) { create(:pacbio_sequel_run, smrt_link_version: versions.first) }
+      let(:plate) { run.plates.first }
```

Comment:
How come this is invalid?

Reply:
Not sure what you mean? The context is invalid which is a bunch of tests for smrt link versioning. The only change here was generating the run as a single entity. It was more of a tidy up.